src/Entities/ExpOrb.h

```diff
@@ -44,9 +44,14 @@ class cExpOrb :
 
 	// tolua_end
 
+	/** Split reward into small values */
+	static void Split(int a_Reward, std::vector<int> & a_SplitReward);
+
 protected:
 	int m_Reward;
 
 	/** The number of ticks that the entity has existed / timer between collect and destroy; in msec */
 	std::chrono::milliseconds m_Timer;
+
+	bool MoveToPlayer;
```

Comment:
Member variables should have the `m_` prefix.
Additionally, the name should indicate `bool`-ness (`MoveToPlayer` is a command, `ShouldMoveToPlayer` or `IsMovingToPlayer` is a property)

Reply:
This member may be unneeded after all, see later comments.